desktop/renderer/remote-component-helper.js

```diff
@@ -17,7 +16,7 @@ export function autoResize () {
           // Height of remote component + offset from parent + top/bottom border
           const originalResizableState = browserWindow.isResizable()
           browserWindow.setResizable(true)
-          browserWindow.setContentSize(browserWindow.getSize()[0], element.scrollHeight + 2 * element.offsetTop + 2 * globalStyles.windowBorder.borderWidth)
```

Comment:
i removed this from the style guide and my grep didn't find this in desktop so i just removed it (old value was 1 so it did nothing). this is mostly a leftover from when we had our own frameless window